test/locations/service/LocationsService.spec.ts

```diff
@@ -522,7 +547,58 @@ describe('Locations Service', () => {
       expect(actualSLAResponse.operatingDays[1].updatedAt).to.equal(expectedSLA.operatingDays[1].updatedAt);
 
     });
+    it('And valid location SLA information update request ' +
+    'When update location SLA information then update location SLA information and ' +
+    'return resolved promise with location SLA information', async () => {
+
+      const response = {
+        status: 200,
+        config: {
+          method: 'PUT'
+        }
+      };
+      const clientPutStub: sinon.SinonStub = sandbox.stub(client, 'put')
+        .yields(null, response, expectedLocationSLAUpdate);
+
+      const locationsService: LocationsService = new LocationsService(client);
+      const actualSLAResponse = await locationsService.updateSla('location-id', expectedLocationSLAUpdate);
+      expect(clientPutStub.args[0][0]).to
+      .equal(`/${Version.V1}${Resource.LOCATIONS}/${expectedLocationSLAUpdate.locationId}/sla`);
+      expect(actualSLAResponse.fulfillmentSLAHours).to.equal(expectedLocationSLAUpdate.fulfillmentSLAHours);
+      expect(actualSLAResponse.operatingDays[0].day).to.equal(expectedLocationSLAUpdate.operatingDays[0].day);
+      expect(actualSLAResponse.operatingDays[0].end).to.equal(expectedLocationSLAUpdate.operatingDays[0].end);
+      expect(actualSLAResponse.operatingDays[0].fulfillmentCutoffTime).to
+        .equal(expectedLocationSLAUpdate.operatingDays[0].fulfillmentCutoffTime);
+      expect(actualSLAResponse.operatingDays[0].id).to.equal(expectedLocationSLAUpdate.operatingDays[0].id);
+      expect(actualSLAResponse.operatingDays[0].open).to.equal(expectedLocationSLAUpdate.operatingDays[0].open);
+      expect(actualSLAResponse.operatingDays[0].createdAt).to
+        .equal(expectedLocationSLAUpdate.operatingDays[0].createdAt);
+      expect(actualSLAResponse.operatingDays[0].updatedAt).to
+        .equal(expectedLocationSLAUpdate.operatingDays[0].updatedAt);
+
+    });
+    it('And exception is thrown ' +
+    'When updating location SLA then return rejected promise with errors', async () => {
+
+      const expectedError = {
+        stack: 'some-error'

```

Comment:
instead of `some-error` use `expectedChannelApeErrorResponse` and check of `404` error.